src/pip/_internal/operations/build/wheel.py

```diff
@@ -24,15 +25,30 @@ def build_wheel_pep517(
     try:
         logger.debug('Destination directory: %s', tempd)
 
+        artifact = "editable" if editable else "wheel"
         runner = runner_with_spinner_message(
-            f'Building wheel for {name} (PEP 517)'
+            f'Building {artifact} for {name} (PEP 517)'
```

Comment:
Is it worth saying `Building editable for <name> (PEP 660)` rather than `(PEP 517)` just to make it clear its using the PEP 660 editable mechanism?